browser/components/shared/title/Title.js

```diff
@@ -0,0 +1,20 @@
+import styled from "styled-components";
+import PropTypes from 'prop-types';
+
+function Title(props) {
+    return (
+        <Root>
+            {props.text}
+        </Root>
+    );
+}
+Button.propTypes = {
+    text: PropTypes.string,
+}
+
+const Root = styled.h4`
+    font-size: 40px;
+    color: ${props => props.theme.colors.dialogueTitle};
+    font-family: ${props => props.theme.fonts.title};
```

Comment:
Try to use `font` instead of `font-family` - it should be possible to include the font-size from the theme

Reply:
Is it normal practice to use `font` css parameter?

Reply:
Yes, the font property is a shorthand property for:

font-style
font-variant
font-weight
font-size/line-height
font-family